examples/official-storybook/stories/addon-resources.stories.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+const myButton = (label, className) => () => (
+  <button type="button" className={className}>
+    {label}
+  </button>
+);
+
+const myIcon = (label, className) => () => (
+  <div>
+    <i className={className} /> {label}
+  </div>
+);
+
+storiesOf('Addons|Resources', module)
+  .add('Primary Large Button', myButton('Primary Large Button', 'btn btn-lg btn-primary'))
```

Comment:
Why do you use this strange components syntax? I didn't see it in use in the other stories... I think we need to be consistent with the examples.